source/19-Psalms.usfm.db

```diff
@@ -2997,7 +2997,7 @@
 \q2 and abandon their wealth to others.
 \q
 \v 11 The grave is their everlasting home,
-\q2 the place they shall live in for ever and ever,
+\q2 the place they shall live in [us:forever|cth:for ever] and ever,
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".